src/App.tsx

```diff
@@ -21,6 +30,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={(event) => {
+              setItemsPerPage(Number(event.target.value));
+              setCurrentPage(1);
+            }}
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here

check other places